### PR TITLE
[BIT-623] Add ValidatorPruneLen & ValidatorLogitsDivergence

### PR DESCRIPTION
**Pull request by opentaco**
### [BIT-623] Add ValidatorPruneLen & ValidatorLogitsDivergence

ValidatorPruneLen (u64) is the amount of tokens to randomly prune from the context in the validator.

ValidatorLogitsDivergence is a u64 value that can be used a penalty scale or threshold for logits divergence during validation.

[BIT-623]: https://opentensor.atlassian.net/browse/BIT-623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Eugene-hu — APPROVED** [review]

**Review by camfairchild — APPROVED** [review]
LGTM 🔥
